Doc/library/2to3.rst

```diff
@@ -351,20 +351,20 @@ and off individually.  They are described here in more detail.
 
    Converts calls to various functions in the :mod:`operator` module to other,
    but equivalent, function calls.  When needed, the appropriate ``import``
-   statements are added, e.g. ``import collections``.  The following mapping
+   statements are added, e.g. ``import collections.abc``.  The following mapping
    are made:
 
-   ==================================  ==========================================
+   ==================================  =============================================
    From                                To
-   ==================================  ==========================================
+   ==================================  =============================================
    ``operator.isCallable(obj)``        ``hasattr(obj, '__call__')``
```

Comment:
Sorry for off-topic, but what is the reason of using ``hasattr(obj, '__call__')`` instead of just ``callable(obj)``? The latter is much faster and looks better IMO.

Reply:
Do you mind to open an issue for this @ilevkivskyi? If ignore Python 3.0 and 3.1, `callable(obj)` looks better replacement.